### DISCUSSION TITLE
OP Goerli will be fully decommissioned in JAN 2024. Migrate to OP Sepolia now for any testing and development needs!

### DISCUSSION THREAD

**Discussion by sbvegan** [ethereum-optimism/developers#107] [Announcements]
Optimism is in the process of migrating testnets from OP Goerli to OP Sepolia. The Goerli testnet [has been deprecated by the Ethereum community](https://github.com/eth-clients/goerli?ref=blog.oplabs.co) since January 2023, and it is only being maintained until the end of 2023.

OP Sepolia is the sustainable path forward, and we encourage all apps and developers to migrate to OP Sepolia for any testing and development needs ASAP.

This is the [network information](https://docs.optimism.io/chain/networks) to include:

- Network Name - OP Sepolia
- RPC URL - https://sepolia.optimism.io/
- Chain ID - 11155420
- Currency Symbol - SepoliaETH
- Block Explorer URLs - https://sepolia-optimism.etherscan.io/ and https://optimism-sepolia.blockscout.com/
- Contract Addresses: https://docs.optimism.io/chain/addresses#testnet-sepolia

End users looking to experiment in a testnet environment should begin to do so on OP Sepolia instead of OP Goerli. Testnet funds can be claimed through the [Superchain faucet](https://app.optimism.io/faucet?ref=developer-forum), up to 1 test ETH per day.

[The bridge](https://app.optimism.io/bridge?ref=developer-forum) supports Sepolia testnets as well. Metamask already supports OP Sepolia so it can be found in the drop down menu. If you need to add it as a custom network somewhere else, please use the network information included above.

If you need support while migrating to OP Sepolia, please comment below and we will provide support.

**Comment by sbvegan** [comment]
Our developer infrastructure team has committed to keeping OP Goerli running until January 7th 2024.

**Reply by idlee695** [reply]
That's great
